### ISSUE TITLE
Sqlite binary unavailable

### ISSUE THREAD

**Issue by d4rken** [d4rken/sdmaid-public#2154]
```
Device K5 (A6020a40) Manufacturer Lenovo Device type Phone Device language Polish
CPU make Qualcomm CPU model MSM8929 Native platform armeabi-v7a, armeabi, arm64-v8a RAM (MB) 2048
Screen size 1080 × 1920 Screen density (DPI) 320 OpenGL ES version 3.0 OS Android 5.1
```

```
Device Ilium X210 (Ilium_X210) Manufacturer Lanix Device type Phone Device language Spanish
CPU make Mediatek CPU model MT6580 Native platform armeabi-v7a, armeabi RAM (MB) 512
Screen size 480 × 800 Screen density (DPI) 240 OpenGL ES version 2.0 OS Android 5.1
```

**Comment by d4rken** [comment]
#2042

**Comment by d4rken** [comment]
```
11-29 14:33:35.986 3536-3574/eu.thedarken.sdm V/SDM:RXS:Harvester:Error: WARNING: linker: /data/data/eu.thedarken.sdm/files/sqlite3_sdm: unused DT entry: type 0x6ffffef5 arg 0xcb4
11-29 14:33:35.986 3536-3574/eu.thedarken.sdm V/SDM:RXS:Harvester:Error: WARNING: linker: /data/data/eu.thedarken.sdm/files/sqlite3_sdm: unused DT entry: type 0x6ffffffe arg 0xdb8
11-29 14:33:35.986 3536-3574/eu.thedarken.sdm V/SDM:RXS:Harvester:Error: WARNING: linker: /data/data/eu.thedarken.sdm/files/sqlite3_sdm: unused DT entry: type 0x6fffffff arg 0x2
11-29 14:33:35.986 3536-3574/eu.thedarken.sdm V/SDM:RXS:Harvester:Error: CANNOT LINK EXECUTABLE: empty/missing DT_HASH in "/data/data/eu.thedarken.sdm/files/sqlite3_sdm" (built with --hash-style=gnu?)
```

**Comment by d4rken** [comment]
Seems like a Android 5.1 specific issue, `hash-style=gnu` support since api 23.

https://android.googlesource.com/platform/bionic/+/android-n-mr2-preview-1/android-changes-for-ndk-developers.md

```
The GNU hash style available with --hash-style=gnu allows faster symbol lookup and is now supported by the dynamic linker in API 23 and above. (Use --hash-style=both if you want to build code that uses this feature >= Android M but still works on older releases.)
```

**Comment by d4rken** [comment]
```
Applet test passed (binary=AppletBinary(type=INTERNAL, envvar=V182F9E04DEEF, path=/data/data/eu.thedarken.sdm/files/sqlite3_sdm, version=3.25.3 2018-11-05 20:37:38 89e099fbe5e13c33e683bef07361231ca525b88f7907be7092058007b75036f2), compat=USER)
```

**Comment by d4rken** [comment]
Fixed with cd6c728

**Comment by d4rken** [comment]
> Still getting this issue. Just bought pro and still getting it. Do you want me to open a new bug?

Yes please. New ticket and provide a debug log.
